### PR TITLE
History of days/hours - moved non-operation parms to command line

### PR DESCRIPTION
**Pull request by gauthig**
Created and added all changes to CHANGELOG.md

1. history days, reset database moved from json config to command line
2. Added -h / --help to show command line parameters
3. History now loads Days and Hours, up to 720 days has been tested
4. added argparse to requirements.txt  and setup.sh
5. Put version number, date and other program meta data in format to later be auto generated by github on new releases
6. Added more debug information using the info() function, but must have -v or --verbose added to command line

Issues
Noticed some points not storing when influxdb container is being backup.  Will look at feature for next release that daily queries for missing datapoints (minutes, hours, days) and does a call specify for them.  Also will be useful for a program or serv er crash as it will check on restart for missing data points.

### CODE REVIEW

**Comment by jertel** [comment]
This is looking good. I made some code and grammar/spelling corrections to your branch. I will need more time to conduct my own testing. Thanks for all your efforts!

**Comment by gauthig** [comment]
Thanks, look forward to your review and feedback.

File: vuegraf.json.sample [hunk 1]
```diff
@@ -8,8 +8,11 @@
         "reset": false,
         "ssl_enable": false,
         "ssl_verify": true
+
     },
     "updateIntervalSecs": 60,
+    "detailedDataEnabled": "true",
+    "summariesDataEnabled": "false",
```
[the remark this answers is not in the document]
Comment (gauthig) [reply vuegraf.json.sample:15]: Left over from earlier version, where I left the history logic for minutes in and gave a choice for summaries.  Not needed anymore

**Review by gauthig — COMMENTED** [review]
Thanks for all the spelling corrections.

**Review by jertel — APPROVED** [review]

**Pull request merged**